pinot-core/src/main/java/org/apache/pinot/core/plan/FilterPlanNode.java

```diff
@@ -207,20 +219,20 @@ private BaseFilterOperator constructPhysicalOperator(FilterContext filter, int n
               // Consuming segments: When FST is enabled, use AutomatonBasedEvaluator so that regexp matching logic is
               // similar to that of FSTBasedEvaluator, else use regular flow of getting predicate evaluator.
               if (dataSource.getFSTIndex() != null) {
-                predicateEvaluator =
-                    FSTBasedRegexpPredicateEvaluatorFactory.newFSTBasedEvaluator((RegexpLikePredicate) predicate,
-                        dataSource.getFSTIndex(), dataSource.getDictionary());
+                predicateEvaluator = FSTBasedRegexpPredicateEvaluatorFactory
+                    .newFSTBasedEvaluator((RegexpLikePredicate) predicate, dataSource.getFSTIndex(),
```

Comment:
we dont need to change the format, right?

Reply:
oh. let me revet this change.